app/src/main/java/org/fossasia/openevent/app/core/orders/detail/OrderDetailFragment.java

```diff
@@ -85,6 +106,29 @@ public void onStop() {
         refreshLayout.setOnRefreshListener(null);
     }
 
+    private void setupRecyclerView() {
+        orderAttendeesAdapter = new OrderAttendeesAdapter();
+
+        RecyclerView recyclerView = binding.attendeesRecyclerView;
+        recyclerView.setLayoutManager(new LinearLayoutManager(context));
+        recyclerView.setAdapter(orderAttendeesAdapter);
+        recyclerView.setItemAnimator(new DefaultItemAnimator());
+
+        Paint paintGreen = new Paint();
+        Paint paintRed = new Paint();
+        paintGreen.setColor(getResources().getColor(R.color.light_green_500));
+        paintRed.setColor(getResources().getColor(R.color.red_500));
+
+        Bitmap doneIcon = BitmapFactory.decodeResource(getResources(), R.drawable.done);
+        Bitmap closeIcon = BitmapFactory.decodeResource(getResources(), R.drawable.close);
+
+        SwipeController swipeController = new SwipeController(orderDetailViewModel, orderAttendeesAdapter,
```

Comment:
No need to create them here, this is a violation of Single Responsibility Principle

Pass context and create in constructor. I meant that don't store the context in field, just use in constructor